CMakeLists.txt

```diff
@@ -73,8 +72,10 @@ include(ROCMClients)
 
 rocm_setup_version(VERSION "0.7")
 
+execute_process(COMMAND ${CMAKE_INSTALL_PREFIX}/bin/mygpu OUTPUT_VARIABLE CURRENT_GPU)
+
 rocm_check_target_ids(DEFAULT_AMDGPU_TARGETS
-  TARGETS "gfx908:xnack-;gfx90a:xnack-;gfx90a:xnack+" )
```

Comment:
Is this xnack flag not required anymore?

Reply:
The build pipes are failing with the message : 

Unsupported GPU; Cannot build tests/samples


This is incorrect... I believe the xnack flags are necessary for the build to be successful